services-gateway-client-transport/src/main/java/io/scalecube/services/gateway/transport/KeepaliveSender.java

```diff
@@ -0,0 +1,23 @@
+package io.scalecube.services.gateway.transport;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KeepaliveSender implements Runnable {
```

Comment:
No need in this class. 
You can implement Runnable right in the WebsocketGatewayClient, and use it in [this block ](https://github.com/scalecube/scalecube-gateway/pull/95/files#diff-d4a33ef8f33409a906bd691f63f4d8d7R152-R155).